### PR TITLE
React capstone project

### PR DESCRIPTION
**Pull request by melaniesigrid**
Hello, and thank you for reviewing my project! On this PR...

- Create a single-page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- I filter the data that you retrieve from the API using a Filter stateless component, using the useParams() hook to take the id of the card being clicked and filter to give the correct details.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project is deployed.

### CODE REVIEW

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @melaniesigrid :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :medal_military: 
- Nice design :heavy_check_mark: 
- Your linters are passing :heavy_check_mark: 
- Nice commit history :+1: 
- Professional README file :heavy_check_mark: 

## Required Changes ♻️

- [x] As this is a mobile web app I think the mobile version should match the provided design please fix your navbar setting and microphone icons style :point_down: 

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,56 +1,48 @@
...
+Clone this repository by typing `git clone https://github.com/melaniesigrid/studio_ghibli.git` on your terminal.
 
 ### Install
+Run `npm install` on your command line.
 
 ### Usage
+Navigate to the project directory `cd studio_ghibli`, and run `npm start`. Or, you may view the live demo here: https://melaniesigrid.github.io/studio_ghibli
 
-### Run tests
-
```
[the remark this answers is not in the document]
Comment (melaniesigrid) [reply README.md:36]: Thank you very much for your useful suggestions and for reviewing my project! I have implemented the changes. ⭐

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @melaniesigrid 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hiromataba) [reply README.md:36]: You are welcome @melaniesigrid :wink:

**Pull request merged**